WordPress/Classes/ViewRelated/Reader/Controllers/ReaderPostActions/ReaderPostMenu.swift

```diff
@@ -154,10 +154,10 @@ struct ReaderPostMenu {
         guard let topic else {
             return false
         }
-        return ReaderHelpers.isTopicTag(topic) ||
-            ReaderHelpers.topicIsDiscover(topic) ||
-            ReaderHelpers.topicIsFreshlyPressed(topic) ||
-            ReaderHelpers.topicIsFollowing(topic)
+        if ReaderHelpers.isTopicTag(topic) {
+            return true
+        }
+        return (topic is ReaderDefaultTopic) && !ReaderHelpers.topicIsLiked(topic)
```

Comment:
There has to be a better way to do it, but I'm not refactoring it in the scope of this PR.